src/test/scala/mesosphere/mesos/PersistentVolumeMatcherTest.scala

```diff
@@ -108,6 +102,7 @@ class PersistentVolumeMatcherTest extends MarathonSpec with GivenWhenThen with M
 
   class Fixture {
     def makeTask(appId: PathId) = MarathonTestHelper.mininimalTask(appId)
+    def makeTask(appId: PathId, reservation: ReservationWithVolumes) = MarathonTestHelper.minimalReservedTask(appId, reservation)
```

Comment:
I would get rid of this forwarding methods... I just need to hold more info in my short term memory to understand the tests (`makeTask(appId) = MarathonTestHelper.minimalTask(appId)`)